DeepSpeech.py

```diff
@@ -8,356 +8,24 @@
 log_level_index = sys.argv.index('--log_level') + 1 if '--log_level' in sys.argv else 0
 os.environ['TF_CPP_MIN_LOG_LEVEL'] = sys.argv[log_level_index] if log_level_index > 0 and log_level_index < len(sys.argv) else '3'
 
-import datetime
```

Comment:
Modularizing this file is a great thing. But why not doing it in a separate PR? I never got a similar refactoring merged due to an overloaded PR.

Reply:
That's why I split this into commits, so you can look at each commit individually. I can also make separate PRs for the commits, but that will require making each PR buildable/passing tests, which will be a lot more work.

Reply:
I know - that's also the reason why I don't insist in doing it. :-) Doing this after the fact is much harder and a big waste of time.